src/app_service/common/net_utils.nim

```diff
@@ -2,12 +2,12 @@ import net
 
 # Util function to test if a port is busy
 proc isPortBusy*(port: Port): bool =
-  var sock: Socket
+  result = false
+  let socket = newSocket()
+  defer: socket.close()
+
   try:
-    sock = newSocket()
-    sock.setSockOpt(OptReuseAddr, true)
-    sock.bindAddr(port)
-    sock.close()
-    return false
+    socket.connect("localhost", port)
+    result = true
```

Comment:
Isn't this reversed? Shouldn't it return `false` if it managed to connect?

Reply:
The logic is somewhat reversed here. If we connect it means something was listening on that port, hence the server is busy. And the other case can be deduced.
I'm still performing some tests... But it works.
In our case this logic holds since client (browser plugin) would connect to the port therefore it's sound.

Reply:
aah ok, yeah it's the opposite logic as before.